scrapy/_monkeypatches.py

```diff
@@ -14,6 +14,11 @@
         from urlparse import uses_query
         uses_query.append('s3')
 
+    # Prevent the DeprecationWarning about SafeConfigParser -> ConfigParser
+    import configparser
+    if not getattr(configparser, 'ConfigParser', None):
+        configparser.ConfigParser = configparser.SafeConfigParser
```

Comment:
I think a deprecation warning in tests doesn't worth monkey-patching stdlib

Reply:
Makes sense, but the class used in `scrapy.utils.conf.get_config`, I think we should still do something about it. I'll change monkey-patching for a `try... except ImportError` approach.